src/index.html

```diff
@@ -13,6 +13,49 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars__container">
+      <div class="stars stars--0">
```

Comment:
Please check how class name for block with modifier should look like according to BEM. Here, it's not used as expected for all the `stars` blocks.

Reply:
According to the requirements I have add 6 stars blocks with 5 stars__star elements each, then add stars--0, stars--1, stars--2 ... stars--5 modifiers to the blocks one per each.  This is not what was expected? I didn't understand a bit.

Reply:
The structure is fine, but the syntax of modifiers is not compliant with BEM. In your code blocks have two classes: first one for block, second one for modifier. Whereas according to BEM syntax this should be a single class name. See https://en.bem.info/methodology/quick-start/#modifier for examples.

Reply:
UPD. There's a kind of discrepancy between the basic BEM guidelines I'm referring to, and the guidelines used on the course. I'm sorry for the confusion. I'll sort it out with the team and come back to you.